packages/client/modules/teamDashboard/components/TeamTasksHeader/TeamTasksHeader.tsx

```diff
@@ -94,7 +95,8 @@ const Avatars = styled('div')({
   display: 'flex',
   paddingTop: 12,
   [desktopBreakpoint]: {
-    paddingTop: 0
+    paddingTop: 0,
+    flexGrow: 1
```

Comment:
-1 it looks like this fixes the issue I mentioned in my earlier comment where only a few avatars would show up on desktop, but on mobile/tablet, I only see three avatars (including overflow) when there's space for many more

Reply:
You are absolutely right. I mentioned here https://github.com/ParabolInc/parabol/pull/6366#issuecomment-1103866909 that there is a controversial thing and we need to agree on how to deal with it.